heal-remedy.lic

```diff
@@ -47,7 +47,7 @@ def wounds
   wounds = DRCH.check_health['wounds']
   pause 1
   echo wounds if $debug_mode_hr
-  hurt = wounds.select { |level, _| level >= $heal_level }.values.flatten
+  hurt = wounds.select { |level, _| level >= $heal_level }.values.flatten.map(&:body_part).uniq
```

Comment:
Updates to new method name and data structure of return type.  `DRCH.check_health['wounds']` are now instances of `DRCH::Wound` class and one of its properties is `body_part`.